### PR TITLE
[CfgEditor] Rename variables

### PR DESCRIPTION
**Pull request by llFreetimell**
This commit renames some of variables for futher features.

ONE-vscode-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

---

Parent Issue : #498 
Draft : #496

### CODE REVIEW

**Review by YongseopKim — APPROVED** [review]
LGTM

**Comment by hyunsik-yoon** [comment]
Could you explain the reason of this change? :-)

**Comment by llFreetimell** [comment]
> Could you explain the reason of this change? :-)

This PR is for reducing duplicated codes :D
Reducing duplicated codes is always good, but there is one more reason... the number of function :(
If I apply this PR, I can expect following things!
```javascript
//// Before
function disableImport() {
}
function disableOptimize() {
}
function disableQuantize() {
}
// ...

//// After
function disableStep(name) {
}
```

File: src/CfgEditor/CfgEditorPanel.ts [hunk 1]
```diff
@@ -242,28 +242,28 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
...
+                    <div class="vscode-panel-view">
+                      <vscode-text-field id="PBInputArrays" startIcon=true size=50 placeholder="semi-colon separated input names">
                         Input Arrays
                         <span slot="end" class="codicon codicon-question" style="cursor: pointer"></span>
                       </vscode-text-field>
                     </div>
-                    <div id="TFOutputArrays" class="vscode-panel-view">
-                      <vscode-text-field startIcon=true size=50 placeholder="semi-colon separated output names">
+                    <div class="vscode-panel-view">
+                      <vscode-text-field id="PBOutputArrays" startIcon=true size=50 placeholder="semi-colon separated output names">
```
Comment (jyoungyun) [inline src/CfgEditor/CfgEditorPanel.ts:260]: (Q) What is the meaning of `PB` prefix?

**Review by jyoungyun — APPROVED** [review]
LGTM

File: src/CfgEditor/CfgEditorPanel.ts (hunk 1 above)
Comment (llFreetimell) [reply src/CfgEditor/CfgEditorPanel.ts:260]: `pb of `model.pb` :)

**Pull request merged**